src/nomad_simulations/schema_packages/model_system.py

```diff
@@ -296,6 +296,32 @@ class Cell(GeometricSpace):
         """,
     )
 
+    def _check_positions(self, positions_1, positions_2) -> list[int, int]:
+        # Check that all the `positions`` of `cell_1` match with the ones in `cell_2`
+        check_positions = []
+        for i1, pos1 in enumerate(positions_1):
+            for i2, pos2 in enumerate(positions_2):
+                if np.allclose(
+                    pos1, pos2, atol=configuration.equal_cell_positions_tolerance
+                ):
+                    check_positions.append([i1, i2])
+                    break
+        return check_positions
+
+    def __eq__(self, other: 'Cell') -> bool:
```

Comment:
you would also define `__ne__` which is simply not `self.__eq__`

Reply:
You mean `return not self.__eq__` in the __ne__?

Reply:
yes indeed, this is required when doing !=